lib/index.js

```diff
@@ -29,12 +40,17 @@ function createListrOptions (options) {
 }
 
 export default function runContentfulExport (params) {
-  summary.startTime = moment()
-
+  const summary = {}
+  const log = []
   const options = parseOptions(params)
 
   const listrOptions = createListrOptions(options)
 
+  // Setup custom error listener to store errors for later
+  setupLogging(log)
+
+  console.log('This is a Beta release')
```

Comment:
the beta notice should be in the yargs `epilog` in [here](https://github.com/contentful/contentful-export/blob/master/lib/usageParams.js#L79)